helm/chart/maesh/values.yaml

```diff
@@ -16,6 +16,7 @@ controller:
       mem: "50Mi"
       cpu: "100m"
   logging:
+    excludeNamespaces:
```

Comment:
It should not be under `controller.logging`

Reply:
Moved